src/pytest_inmanta_lsm/plugin.py

```diff
@@ -132,10 +135,40 @@ def remote_orchestrator_container(
         yield None
         return
 
+    orchestrator_image = inm_lsm_ctr_image.resolve(request.config)
+    latest_compose_file = pathlib.Path(__file__).parent / "resources/docker-compose.yml"
+    legacy_compose_file = pathlib.Path(__file__).parent / "resources/docker-compose-iso6.yml"
+    try:
+        compose_file = inm_lsm_ctr_compose.resolve(request.config)
+    except pytest_inmanta.test_parameter.ParameterNotSetException:
+        # The compose file is not set, we can then either use the default one, or
+        # the default legacy one (for <iso7.1).  To decide which one is the most
+        # appropriate, we parse the container image tag and extract the iso version
+        iso_major_version_match = re.fullmatch(
+            r"(?:.*)\/service-orchestrator:((\d+(?:\.\d+)*)(?:\-dev)?|dev)",
+            orchestrator_image,
+        )
+        if not iso_major_version_match:
+            LOGGER.warning(
+                "Can not parse orchestrator image tag: %s.  Using legacy docker-compose file",
+                orchestrator_image,
```

Comment:
It's not clear to my why we'd default to legacy? All our containers should match this format, shouldn't they?

Reply:
Ours, yes, but nothing prevents the user of pytest-inmanta-lsm to run the tests against a custom image, this is what this case is for.  I added a comment